Telescope crashes at job failure

**What happened**:

Telescope crashes when processing one of the jobs

```
telescope_1  | [ 2020-02-07 14:49:41.081 ] DEBUG (19 on 7415bd343846): Job 82af89b3d4 is active
redis_1      | 1:S 07 Feb 2020 14:49:41.182 * Before turning into a replica, using my master parameters to synthesize a cached master: I may be able to synchronize with the new master with just a partial transfer.
redis_1      | 1:S 07 Feb 2020 14:49:41.182 * REPLICAOF 82.118.17.133:8887 enabled (user request from 'id=8 addr=159.65.147.175:45258 fd=13 name= age=1 idle=0 flags=N db=0 sub=0 psub=0 multi=-1 qbuf=47 qbuf-free=32721 obl=0 oll=0 omem=0 events=r cmd=slaveof')
telescope_1  | [ 2020-02-07 14:49:41.213 ] INFO  (30 on 7415bd343846): [HTTP 200 - OK] Feed has new content: https://medium.com/feed/@jrkong.hfd
telescope_1  | [ 2020-02-07 14:49:41.848 ] ERROR (19 on 7415bd343846): Job 82af89b3d4 failed.
telescope_1  |     error: {
telescope_1  |       "command": {
telescope_1  |         "name": "evalsha",
telescope_1  |         "args": [
telescope_1  |           "cf5b7596b81a84d57147a584a36084cc6c578c70",
telescope_1  |           "6",
telescope_1  |           "bull:feed-queue:active",
telescope_1  |           "bull:feed-queue:completed",
telescope_1  |           "bull:feed-queue:82af89b3d4",
telescope_1  |           "bull:feed-queue:wait",
telescope_1  |           "bull:feed-queue:priority",
telescope_1  |           "bull:feed-queue:active@3543e184-c9b4-4e27-a537-45fca8972edc",
telescope_1  |           "82af89b3d4",
telescope_1  |           "1581086981844",
telescope_1  |           "returnvalue",
telescope_1  |           "[{\"id\":\"d697c367f4\",\"author\":\"Alex Kong\",\"title\":\"ESLint for Mozilla 
...
telescope_1  |           "3543e184-c9b4-4e27-a537-45fca8972edc",
telescope_1  |           "1",
telescope_1  |           "{\"jobId\":\"82af89b3d4\",\"val\":\"[{\\\"id\\\":\\\"d697c367f4\\\",\\\"author\\\":\
...
/@jrkong.hfd?source=rss-a38be42052fa------2\\\",\\\"guid\\\":\\\"https://medium.com/p/f61ceaeee28e\\\"}npm ERR! code ELIFECYCLE
telescope_1  | npm ERR! errno 1
telescope_1  | npm ERR! telescope@0.0.1 start: `node src/backend`
telescope_1  | npm ERR! Exit status 1
telescope_1  | npm ERR! 
telescope_1  | npm ERR! Failed at the telescope@0.0.1 start script.
telescope_1  | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
telescope_1  | 
telescope_1  | npm ERR! A complete log of this run can be found in:
telescope_1  | npm ERR!     /root/.npm/_logs/2020-02-07T14_49_41_879Z-debug.log
```

**What should have happened**:

Telescope should keep running

**How to reproduce it (as precise as possible)**:

Run telescope for about 15 min

**Environment**:

- OS: Fedora 30

--- a/src/backend/index.js
+++ b/src/backend/index.js
@@ -103,9 +103,10 @@ feedQueue.on('drained', loadFeedsIntoQueue);
  * If there is a failure in the queue for a job, set the feed to invalid
  * and save to Redis
  */
-feedQueue.on('failed', job =>
+// Commenting out below for now as it is suspected to be crashing staging server
+/* feedQueue.on('failed', job =>
   invalidateFeed(job.data).catch(error => logger.error({ error }, 'Unable to invalidate feed'))
-);
+); */
 
 /**
  * Also load all feeds now and begin processing.

--- a/src/backend/index.js
+++ b/src/backend/index.js
@@ -47,14 +47,16 @@ async function updateFeed(feed) {
  * Invalidates a feed
  * @param feedData - Object containing feed data
  */
-async function invalidateFeed(feedData) {
+// Commented out as of now as it is breaking the staging server. Uncomment
+// this block and 109 to enable invalidating feeds
+/* async function invalidateFeed(feedData) {
   const feed = Feed.parse(feedData);
   await feed.setInvalid(feedData.reason || 'unknown reason');
   logger.info(
     `Invalidating feed ${feedData.url} for the following reason: ${feedData.reason ||
       'unknown reason'}`
   );
-}
+} */
 
 /**
  * Process all of these Feed objects into Redis and the feed queue.

